src/main/kotlin/avans/avd/exceptions/MissingRoleException.kt

```diff
@@ -0,0 +1,5 @@
+package avans.avd.exceptions
+
+import avans.avd.users.Role
+
+class MissingRoleException(expectedRoles: List<Role> = emptyList()) : IllegalAccessException("Missing role(s): ${expectedRoles.joinToString { it.name }}")
```

Comment:
I replaced the guard clauses in the routes with exceptions to adhere to more conventional programming style, especially since the `return@get call.respond(...)` syntax can be quite hard to read for newcomers.